<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
  <meta name="author" content="Sergio Peignier">
  <title>Sergio-Peignier-Home</title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<h1 id="lateral-gene-transfer-and-the-nature-of-bacterial-innovation"><a href="http://www.nature.com/nature/journal/v405/n6784/abs/405299a0.html">Lateral gene transfer and the nature of bacterial innovation</a></h1>
<ul>
<li>Regions adjacent to genes identified as being HGT often contain vestiges of the sequences affecting their integration:
<ul>
<li>translocatable elements</li>
<li>transfer origins of plasmids</li>
<li>known attachment sites of phage integrases</li>
</ul></li>
<li>HGT may act upon homologous sequences
<ul>
<li>However, the action of mismatch correction systems greatly reduces the efficiency of homologous recombination when donor and recipient sequences contain nonidentical bases12,13.</li>
<li>homologous recombination is most successful in integrating DNA into the chromosome when the donor and recipient are relatively closely related.</li>
<li>principally affects the variation in existing genes, rather than introducing unique traits to the genome</li>
<li>role in the ecological and physiological diversification of bacteria is apt to be negligible.</li>
</ul></li>
<li>Conditions of acquisition of HGT
<ul>
<li>there needs to be a means for the donor DNA to be delivered into the recipient cell.</li>
<li>the acquired sequences must be incorporated into the recipientâ€™s genome (or become associated with an autonomous replicating element).</li>
<li>the incorporated genes must be expressed in a manner that benefits the recipient microorganism.</li>
</ul></li>
<li>Mechanisms
<ul>
<li>transformation:
<ul>
<li>uptake of naked DNA from the environment</li>
<li>potential to transmit DNA between very distantly related organisms.</li>
<li>Certain bacterial species are perpetually competent to accept DNA, whereas others become competent upon reaching a certain physiological stage in their life cycle</li>
<li>Some species require specific recognition sequences which are present in their respective genomes at frequencies far greater than those expected at random</li>
<li>many of the naturally competent bacterial species do not display sequence preference and are capable of high levels of transformation<br />
</li>
</ul></li>
<li>transduction:
<ul>
<li>By bacteriophage that has replicated within a donor microorganism and packaged random DNA fragments.</li>
<li>Amount of DNA that can be transferred in a single event is limited by the size of the phage capsid.</li>
<li>Host needs to be recognized by the bacteriophage.</li>
<li>Delivery of double-stranded DNA into the recipient cytoplasm, and promotion the integration of DNA into the chromosome and protect the transferred sequences from degradation by host restriction endonucleases.</li>
</ul></li>
<li>conjugation:
<ul>
<li>physical contact between donor and recipient cells</li>
<li>can mediate the transfer of genetic material between life domains (e.g., bacteria and yeast)</li>
<li>DNA is transferred from a donor to a recipient strain by either a self-transmissible or mobilizable plasmid:
<ul>
<li>Most common method of acquiring antibiotic resistance determinants.</li>
<li>rarely integrated into the chromosome</li>
</ul></li>
<li>Or transfer of chromosomal sequences by plasmids that integrate into the chromosome (forming an Hfr strain that can transfer donor DNA to recipient cells).
<ul>
<li>When flacked by insertion sequences, it may be mobilized as a complex transposon and integrate piece of plasmid into the chromosomes.</li>
</ul></li>
<li>and by conjugative transposons, which encode proteins required for their excision from the donor, formation of a conjugative bridge and transposition into the recipient strain: Integrons consist of three elements:
<ul>
<li>attachment site where the horizontally acquired sequence is integrated</li>
<li>gene encoding a site-specific recombinase (that is, integrase); and a promoter that drives expression of the incorporated sequence.<br />
</li>
</ul></li>
</ul></li>
<li>Stably maintained if:
<ul>
<li>persistence as an episome, which requires selection to avoid stochastic loss</li>
<li>homologous recombination, which will serve primarily to reassort variation among closely related taxa and is unlikely to allow introduction of novel traits</li>
<li>integration mediated by bacteriophage integrases or mobile element transposases</li>
<li>illegitimate incorporation through chance double-strand break repair</li>
</ul></li>
</ul></li>
<li><p>HGT pathogenicity islands: large clusters of virulence genes and can, upon incorporation, transform a benign organism into a pathogen.</p></li>
<li><p>We would expect that the successful mobilization of complex metabolic traits requires the physical clustering of genes, such that all necessary genes will be transferred in a single step. As a result, horizontal inheritance will select for gene clusters and for operons, which can be expressed in recipient cells by a host promoter at the site of insertion.</p></li>
</ul>
<h1 id="the-origin-of-new-genes-glimpses-from-the-young-and-old"><a href="http://www.nature.com/nrg/journal/v4/n11/full/nrg1204.html">The origin of new genes: glimpses from the young and old</a></h1>
<h1 id="mechanisms-of-and-barriers-to-horizontal-gene-transfer-between-bacteria"><a href="http://www.nature.com/nrmicro/journal/v3/n9/full/nrmicro1234.html">Mechanisms of, and Barriers to, Horizontal Gene Transfer between Bacteria</a></h1>
<h1 id="horizontal-transfer-not-duplication-drives-the-expansion-of-protein-families-in-prokaryotes"><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1001284">Horizontal Transfer, Not Duplication, Drives the Expansion of Protein Families in Prokaryotes</a></h1>
<ul>
<li>Xenologs : Homologs acquired by horizontal gene transfe</li>
<li>Paralogs: Homologs acquired by intra-chromosomal duplication</li>
<li>vast majority of expansions of protein families are due to transfer, even among large genomes.</li>
<li>Transferred genes (xenologs) persist longer in prokaryotic lineages possibly due to a higher/longer adaptive role. xenologs share very few protein-protein interactions and genetic regulators</li>
<li>Duplicated genes (paralogs) are expressed more, and, when persistent, they evolve slower. paralogs share most protein-protein interactions and genetic regulators</li>
<li>This suggests that gene transfer and gene duplication have very different roles in shaping the evolution of biological systems:
<ul>
<li>transfer allows the acquisition of new functions</li>
<li>duplication leads to higher gene dosage.</li>
</ul></li>
<li>Genes are identical upon duplication and if there is no direct selection on multiple copies, the redundant extra copies are quickly lost.</li>
<li>Gene duplications persist in genomes due to selection for increased gene dosage.</li>
<li>Periods of such adaptive gene duplication are windows of opportunity for the acquisition of new functions by the slow evolutionary divergence between the duplicates [6?8].</li>
<li>Duplicated genes can be fixed because sub or neo-functionalization when selection for higher gene dosage ceases (reviewed in [9?12]).</li>
</ul>
<p>+Horizontal gene transfer (HGT) results in the acquisition of radically new genetic information, liberating genomic evolutionary processes from tinkering exclusively with pre-existing genes [22? 24].</p>
<ul>
<li>HGT occurs at very high rates leading to very large species pan-genomes [25]</li>
</ul>
<p>## Expansions of protein families arise most frequently by HGT</p>
<ul>
<li>Duplication processes in prokaryotes produce tandem, strictly identical copies of genes [8,38,39].</li>
<li>(short horizon evolution) paralogs arising in the lineages are co-localized because of the low rearrangement rates and are highly similar in sequence because of the low mutation rates.</li>
<li>transferred genes are inserted almost randomly in genomes and show a large range of sequence similarity relative to the native homolog.</li>
<li>We find that the majority of gene expansions arrive via transfer not via duplication</li>
</ul>
<h2 id="xenologs-persist-longer-than-paralogs">Xenologs persist longer than paralogs</h2>
<ul>
<li>xenologs have an average age of introduction that is twice that of paralogs</li>
<li>Xenologs are therefore more persistent than paralogs (Table 2).</li>
<li>Overall, these results show that expansions by HGT are more abundant and remain for longer periods of time in genomes.</li>
</ul>
<h2 id="xenologs-evolve-faster">Xenologs evolve faster</h2>
<ul>
<li>Paralogs being more expressed than xenologs, they evolve slower in synonymous positions, due to selection on codon usage bias, and even slower in non-synonymous positions, due to purifying selection on protein sequence.</li>
<li>Less conversion?</li>
</ul>
<h2 id="xenologs-share-fewer-protein-interactions">Xenologs share fewer protein interactions</h2>
<ul>
<li>As similarity increases between the pair of proteins so does the percentage of shared protein interactions (R2 = 0.60, p,0.01).</li>
<li>Highly similar xenologs have DPPI similar to paralogs.</li>
<li>Hence, the differences observed between xenologs and paralogs in terms of their integration in protein-protein interactions reflect the intrinsic consequences of the mechanisms of acquisition:
<ul>
<li>xenologs may or may not be very similar in sequence,</li>
<li>but recent paralogs are necessarily very similar. As a result, PPI networks will evolve very differently in lineages affected by horizontal transfer relative to lineages with strictly vertical transmission of genetic information.</li>
</ul></li>
</ul>
<h2 id="xenologs-share-fewer-regulatory-mechanisms">Xenologs share fewer regulatory mechanisms</h2>
<ul>
<li>Paralogs share more regulators than xenologs</li>
<li>paralogs appear with overlapping regulatory dependencies, whereas transfer generates xenologs that potentially have highly different regulatory dependencies.</li>
<li>This picture is in agreement with the frequently observed co-transfer of transcription factors and their target genes [52], and subsequent slow integration of these sub-networks in the larger genetic network [53] (millions of years: Integration of horizontally transferred genes into regulatory interaction networks takes many million years)</li>
</ul>
<h1 id="the-impact-of-long-distance-horizontal-gene-transfer-on-prokaryotic-genome-size"><a href="http://bioinformatics.bio.uu.nl/pdf/Cordero.pnas09-0.pdf">The impact of long-distance horizontal gene transfer on prokaryotic genome size</a></h1>
<ul>
<li>Horizontal transfer is the dominant force in expanding the gene repertoires of bacterial genomes (30).</li>
<li>Microbial genomes are highly dynamic entities, constantly acquiring and losing genes (30).</li>
<li>polyphyletic genes distributed in a patchy manner among distant species, have superlinear trend with respect to the size of the genome emerges.</li>
<li>Whereas most genes may have originated from transfers, large genomes tend to have a greater contribution of phylogenetically discordant genes.</li>
<li>Strong correlation between the size of the genomes where those polyphyletic genes are found.</li>
<li>Explanations:
<ul>
<li>Intrinsic rate of illegitimate recombination per gene increases in large genomes as a result of the larger number of transposons, integrases, or phage elements that may facilitate the integration of foreign DNA (31?33). In addition, larger genomes tend to be composed of multiple plasmids or megaplasmids, which suggest a higher rate of transmembrane DNA translocation. Positive correlation observed between genome size and the modularity of the biochemical networks (34).</li>
<li>Environmental factors, increases in large genomes: If genome size is correlated to biodiversity and cell density, one can expect that, in average, the amount of DNA available for uptake during the lifetime of a bacterium is larger in large genomes. An attractive hypothesis is that the increase in phylogenetic diversity in the environment and the broadening of gene repertoires in the genome could be a circular process: more complex ecological interactions in species-rich communities could increase the demand for larger gene repertoires, which are expanded by accepting genes from those phylogenetically distant organisms in the environment.</li>
</ul></li>
</ul>
<h1 id="prokaryotic-evolution-in-light-of-gene-transfer"><a href="http://mbe.oxfordjournals.org/content/19/12/2226.full.pdf+html">Prokaryotic Evolution in Light of Gene Transfer</a></h1>
</body>
</html>
