<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
  <meta name="author" content="Sergio Peignier">
  <title>Sergio-Peignier-Home</title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<h1 id="genome-rearrangement-operations-insights-into-structural-variations-and-genome-re-arrangements-in-prokaryotic-genomes">Genome Rearrangement operations <a href="http://bioinformatics.oxfordjournals.org/content/early/2014/09/03/bioinformatics.btu600.full.pdf+html">Insights into structural variations and genome re-arrangements in prokaryotic genomes</a></h1>
<ul>
<li>illegitimate recombination events (homologous recombination)</li>
<li>imprecise non-homologous repair mechanism during aberrant DNA replication to repair broken replication forks</li>
<li>few kb to sometimes up-to millions of bp</li>
<li>Outcomes:
<ul>
<li>loss</li>
<li>amplification (Andersson, et al., 1998)</li>
<li>translocation (Block, et al., 2012)</li>
<li>inversions (Johnson, 1991)</li>
<li>disruption of an existing gene (Jasin and Schimmel, 1984),</li>
<li>creation of a new gene (Nagarajan, et al., 2012)</li>
<li>chimeric gene product through gene fusions (Nogami, et al., 1985; Roth, et al., 1996).</li>
<li>altered gene copy number and thereby affecting its expression (Rebollo, et al., 1988): Deletions and duplications can potentially lead to altered doses of otherwise functionally intact elements</li>
</ul></li>
<li>Types:
<ul>
<li>Deletion</li>
<li>Duplication<br />
</li>
<li>Inversion</li>
<li>Translocation</li>
<li>Insertion</li>
</ul></li>
<li>Molecular predisposition:
<ul>
<li>chromosomal contexts:
<ul>
<li>sequence and structural motifs,</li>
<li>repeat elements,</li>
<li>insertion sequence (IS) elements</li>
<li>transposon elements (TE) (Mahillon and Chandler, 1998; Treangen, et al., 2009).</li>
</ul></li>
<li>Homologous repetitive segments within one chromosome or on different chromosomes can serve as sites for illegitimate crossing-over.</li>
<li>Extensive array of repetitive sequences, recombination hotspots (Aras, et al., 2003; Treangen, et al., 2009).</li>
<li>Generic repeats may arise by Horizontal Gene Transfert whereby the incoming DNA fragment contains the information already present in the host genome and integrates seamlessly into the host genome using site-specific recombination (Treangen, et al., 2009).</li>
</ul></li>
</ul>
<h1 id="duplications">Duplications</h1>
<ul>
<li><a href="https://tel.archives-ouvertes.fr/tel-00482373/file/these.pdf">Caractérisation et analyse évolutive des répétitions intragéniques : une étude au niveau des gènes, des séquences protéiques et des structures tridimensionnelles</a></li>
<li><a href="http://web.stanford.edu/group/hopes/cgi-bin/hopes_test/all-about-mutations/#the-unequal-crossing-over-model">all about mutations</a></li>
<li><a href="http://www.tandfonline.com.sci-hub.cc/doi/full/10.1080/10409230701507757">Adaptative amplification</a></li>
</ul>
<h3 id="polymerase-slippage-model">Polymerase slippage model:</h3>
<ul>
<li>Polymerase slips from the template strand during replication.</li>
<li>It causes the new strand to unpair (release) from the template strand.</li>
<li>The new strand tries to reattach to the template strand, it will have many identical copies of the codon to choose from.</li>
<li>With so many identical codon copies to reattach to, the new strand may reattach to the template at the wrong copy.</li>
<li>Repair mechanisms may lead to deletion or duplication</li>
<li>Small duplications require little energy (for slippage to occure) and are easily accounted for by the slippage model.</li>
<li>However, many of the expansions that researchers see are quite large, sometimes in the hundreds of repeats. Slippage as the cause of these large expansions would be energetically unfavorable.</li>
<li>Can be enhanced by harpin loops (larger duplications and deletions can appear)</li>
</ul>
<h3 id="homologous-recombination">Homologous recombination:</h3>
<ul>
<li>Appears thanks to structure called Amplicon:
<ul>
<li>DNA sequence surounded by primers (repeted sequences)</li>
</ul></li>
<li>3 mechanisms:
<ul>
<li>Unequal crossing over =&gt; tandem repetitions of variable sizes (or deletions)</li>
<li>Excision of replicon and then insertion in the genome (same chromosome or distincct one)</li>
<li>Circular replicon =&gt;
<ul>
<li>double strand break + single gap in one strand</li>
<li>annealing between the portion around the double break and the compelmentary strand in the gap</li>
<li>circle closed by ligation</li>
<li>replication fork stablished in the circle</li>
<li>elongation of sequence in the circle (several duplications)</li>
<li>Circle breaks and strands are linked (at double strand break point)</li>
</ul></li>
</ul></li>
</ul>
<h3 id="transposable-elements">Transposable elements:</h3>
<ul>
<li>2 sort of TE (creates separated duplications):
<ul>
<li>class 1 (RNA dependent):
<ul>
<li>Gene is transcribed</li>
<li>Reverse transcription</li>
<li>2 sort of insertions into the genome:
<ul>
<li>LTR (Long Terminal Repeat), if the TE has repeated sequences siding the sequence, insertion by recombination in zones containing repeated sequences</li>
<li>no-LTR, in this case the TE gets inserted in double strand break points</li>
</ul></li>
</ul></li>
<li>class 2 (DNA dependent):
<ul>
<li>The element is cut from the genome =&gt; double strand break</li>
<li>Paste somewhere else</li>
<li>If another copy is around it can get pasted there (double strand break point)</li>
<li>If there cell under replication the segment can be inserted and transcribed, and clipped, so finally a new free element will exist see this <a href="https://es.wikipedia.org/wiki/Transposón#/media/File:Transposición_no_conservativa.png">link</a></li>
</ul></li>
</ul></li>
</ul>
<h3 id="gene-convertion">Gene convertion</h3>
<ul>
<li>Homologus recombination with replacement of one sequence by another =&gt; correction of a mutated sequece.</li>
</ul>
<h1 id="genesis-effects-and-fates-of-repeats-in-prokaryotic-genomes"><a href="http://femsre.oxfordjournals.org/content/33/3/539.abstract">Genesis, effects and fates of repeats in prokaryotic genomes</a></h1>
<ul>
<li>Cause creation:
<ul>
<li>Intrachromosomal recombination</li>
<li>horizontal transfer</li>
</ul></li>
<li>recombination operations:
<ul>
<li>amplifications,</li>
<li>deletions</li>
<li>rearrangements</li>
</ul></li>
<li>Types: immense diversity of repeats in genomes
<ul>
<li>created by selfish elements</li>
<li>protection against selfish elements</li>
<li>transient gene amplifications</li>
<li>stable duplications.</li>
<li>some repeats do not carry any adaptive value</li>
<li>Allow functional diversification and increased expression.</li>
<li>Number and types of repeats are also quite diverse and in line with ecological variables, such as host-dependent associations or population sizes, and with genetic variables, such as the recombination machinery</li>
</ul></li>
<li>Role:
<ul>
<li>play extremely important roles in genome stability and plasticity.</li>
<li>Through recombination, repeats increase the rates of rearrangement, amplification and deletion of genetic material</li>
<li>Deletions and amplifications of short tandem repeats are particularly frequent</li>
<li>nonreciprocal recombination (gene conversion), repeats can also lead to new combinations of pre-existing alleles.</li>
<li>Finally, closely spaced inverted repeats (IRs) result in structured DNA or RNA molecules. Repeated elements have the potential to increase genome plasticity and have been recruited to produce localizing sequence diversification by recombination processes.</li>
</ul></li>
</ul>
<h3 id="what-is-a-dna-repeat-multiple-perspectives-on-repeats">What is a DNA repeat? Multiple perspectives on repeats</h3>
<ul>
<li>Repeats:
<ul>
<li>Source of functional overlapping:
<ul>
<li>Establish an association between genetic elements</li>
<li>Redundancy: two proteins with a similar protein domain, duplicated genes or operons</li>
<li>similar copies of a repeat in their regulatory regions: associations between genes in genetic networks.</li>
</ul></li>
<li>Source of sequence recombination:
<ul>
<li>mechanisms require some level of sequence similarity between the sequences.</li>
<li>RecA promotes strand exchange between homologous or homeologous sequences</li>
<li>RecA dependent recombination steps:
<ul>
<li>efficiency decreases exponentially with sequence divergence:</li>
</ul></li>
<li>Slipped-strand mispairing
<ul>
<li>Decreases exponentially with the distance between the copies of the repeat (Che ́din et al., 1994; Lovett et al., 1994)</li>
<li>increases with the length of the repeated elements (Peeters et al., 1988; Pierce et al., 1991).</li>
</ul></li>
</ul></li>
</ul></li>
</ul>
<h3 id="the-origin-of-repeats">The origin of repeats</h3>
<h4 id="generic-repeats">Generic repeats</h4>
<ul>
<li>Cause:
<ul>
<li>horizontal gene transfer:
<ul>
<li>creation of repeats whenever the incoming DNA contains elements similar to information already present in the genome.</li>
<li>integration into genome by:
<ul>
<li>site-specific recombination (e.g., phage integrase)</li>
<li>Homologus recombination:
<ul>
<li>if incoming DNA contains homology with the chromosome at both extremities =&gt; double cross-over leads to genetic replacement (Niaudet et al., 1985).</li>
<li>if incoming DNA is circular =&gt; then one single region of similarity with the chromosome allows integration of the genetic element by homologous recombination in a Campbell-like manner (Duncan et al., 1978). =&gt; repeats are created close together as they are spaced by the sequence of the incoming element.</li>
</ul></li>
</ul></li>
</ul></li>
<li>homologous recombination between smaller or more degenerate repeats:
<ul>
<li>Leads to gene conversion:
<ul>
<li>increase the region of similarity between the elements</li>
<li>Creates larger repeats (Santoyo &amp; Romero, 2005)</li>
<li>Maintains the sequence similarity between repetitive elements (Harvey &amp; Hill, 1990) (Hughes, 2000).</li>
</ul></li>
<li>Some sequences escape from gene conversion:
<ul>
<li>The sequence may diverges after a given threshold, it may become free from the homogenizing effect of gene conversion because recombination frequency drops exponentially</li>
</ul></li>
</ul></li>
<li>Illegitimate recombination between potentially very small and close repeats:
<ul>
<li>Illegitimate recombination between small degenerate closely spaced
<ul>
<li>Cause dislocation mutagenesis =&gt; production of a strict repeat from a degenerate one (Schaaper, 1988).</li>
<li>Given slipped-strand mispairing properties =&gt; repeats must be nearby in the genome but they may only be a few nucleotides long. Such repeats can be so small that they are easily found in genetic texts (Karlin &amp; Cardon, 1994).</li>
</ul></li>
<li>amplicon model of repeat generation (Romero &amp; Palacios, 1997)
<ul>
<li>First, illegitimate recombination leads to the creation of short close repeats (slippage events (Edlund &amp; Normark, 1981) ).</li>
<li>Second, these repeats are low-frequency amplicons that may amplify in tandem.</li>
<li>Third, resulting amplifications are large and can expand and contract by homologous recombination (Levinson &amp; Gutman, 1987)</li>
</ul></li>
</ul></li>
</ul></li>
</ul>
<h4 id="selfish-elements">Selfish elements</h4>
<ul>
<li>Transposable Elements (TE):
<ul>
<li>retrotransposons or class I elements:
<ul>
<li>RNA intermediate</li>
</ul></li>
<li>DNA transposons or class II elements:
<ul>
<li>DNA intermediate.</li>
<li>Most frequent and repetitive in prokariots</li>
<li>IS are the most aboundant members
<ul>
<li>Carry one or two ORFs encoding the transposase</li>
<li>generally (but not always) flanked by short IRs (Mahillon &amp; Chandler, 1998).</li>
<li>Composite transposons are elements flanked by IS elements</li>
<li>noncomposite transposons are flanked only by IRs</li>
<li>Transposons may carry unrelated genes such depending on their mechanism of transposition (for a review, see Mahillon &amp; Chandler, 1998).</li>
</ul></li>
<li>TEs move in genome by a process that is independent of DNA homology:
<ul>
<li>cut-and-paste</li>
<li>copy-and-paste : if transposition takes place at the moment of replication from a newly replicated region to a nonreplicated region</li>
</ul></li>
</ul></li>
<li>3 prokaryotic types:
<ul>
<li>Group II introns (Ferat &amp; Michel, 1993; Simon et al., 2008):
<ul>
<li>distributed among all three domains of life</li>
<li>self-splicing. Muliply via reverse transcription<br />
</li>
</ul></li>
<li>retrons (Temin, 1989; Lampson et al., 2005)
<ul>
<li>multicopy single-stranded DNA covalently linked to RNA</li>
<li>reverse transcibed</li>
</ul></li>
<li>diversity-generating retroelements (Medhekar &amp; Miller, 2007).
<ul>
<li>selective advantage by introducing vast amounts of sequence diversity into target genes (Doulatov et al., 2004).</li>
</ul></li>
</ul></li>
</ul></li>
</ul>
<h3 id="the-functional-consequences-of-repeats">The functional consequences of repeats</h3>
<h4 id="gene-genesis-by-duplication-and-divergence">Gene genesis by duplication and divergence</h4>
<ul>
<li>most evolutionary novelty is thought to arise from the duplication and divergence of pre-existing elements (Ohno, 1970; Taylor &amp; Raes, 2004).</li>
<li>Gene duplication creates redundancy, thus freeing the copies from some functional constraints. If the copies are effectively redundant, then one of them is likely to be lost by lack of selection against inactivating mutations</li>
<li>However, in rare cases, the accumulation of mutations may lead either to:
<ul>
<li>neofunctionalization, one of the copies acquires a new function</li>
<li>subfunctionalization, the two copies specialize in subfunctions of the primitive gene (Fig. 6) (Lynch &amp; Katju, 2004)</li>
<li>Both rely heavily on the accumulation of point mutations.</li>
</ul></li>
<li>If there is strong selection for both copies at the time of amplification, then the amplification will be stable and mutational processes could have much more time to diversify the function of the genes (Francino, 2005; Bergthorsson et al., 2007):
<ul>
<li>e.g.,A gene confers a secondary function with weak efficiency</li>
<li>Need for that function increases,</li>
<li>Amplification of the gene is selected</li>
<li>amplified gene accumulates mutations in the different copies.</li>
<li>Mutations in one of the copies of the gene allowing an increase in efficiency of the secondary function become highly adaptive.</li>
<li>Other copies could be deleted or be used for the primary or other function</li>
<li>Leads to subfunctionalization (Francino, 2005).</li>
<li>Might be a frequent event (Patrick et al., 2007).</li>
<li>Bias of deletion over amplification in prokaryotic genomes (Lawrence et al., 2001; Mira et al., 2001). Once repeats are created by recombination, typically in tandem, they are then deleted at high rates</li>
<li>gene conversion homogenize the two copies of the gene =&gt; reduce sequence divergence and slow subfunctionalization and neofunctionalization (Santoyo &amp; Romero, 2005).</li>
<li>large rearrangements tend to be highly deleterious (Rocha, 2008). Repeats can be stabilized if, after creation in tandem, they are physically separated by chromosome rearrangements (Achaz et al., 2000).</li>
<li>CONCLUSION: ample evidence for gene amplifications in genomes, demonstrated by the abundance of repeated elements and some large families of paralogues (Snel et al., 2002; Gevers et al., 2004; Alm et al., 2006; Cho et al., 2007). It is unclear whether gene amplifications last long enough to be responsible for the creation of these large gene families (Lerat et al., 2005), but it is not clear…</li>
</ul></li>
<li>Horizontal Transfert:
<ul>
<li>quick avenue for acquiring functions that have already endured selection in other contexts</li>
<li>ample evidence that it brings new genes into most genomes at very high rates (Hao &amp; Golding, 2006)</li>
</ul></li>
</ul>
<h3 id="intragenic-repeats-and-the-evolution-of-protein-function">Intragenic repeats and the evolution of protein function</h3>
<ul>
<li>Repeats exist inside genes and cause intragenic amplifications of genetic material.</li>
<li>Intragenic amplification =&gt; direct influence of DNA sequences on protein sequences: consequences at the three levels (DNA seq, protein seq, struccture) often deleterious consequences.</li>
<li>Many proteins contain repeated zones.</li>
<li>Several proteins contain the same domains</li>
<li>Domain duplication is one of the main sources of new domains (Vogel et al., 2004).</li>
<li>Tandem repeats can lead to misfolding and are typically avoided by proteins.</li>
<li>Some proteins evolve by amplifying domains (quite rare however).</li>
<li>When repeats are found in tandem, some residues named structural gatekeepers may prevent the proteins from misfolding.</li>
</ul>
<h2 id="the-evolutionary-impact-of-repeats">The evolutionary impact of repeats</h2>
<h3 id="repeats-recombination-and-adaptation">Repeats, recombination and adaptation</h3>
<ul>
<li>Schematically, environmental challenges can be classified according to
<ul>
<li>frequency with which they arise</li>
<li>possibility of building appropriate physiological responses to tackle them.</li>
<li>The evolutionary patterns associated with adaptation depends on both</li>
</ul></li>
<li>Stresses
<ul>
<li>tackled by a physiological response based on the regulated expression of a set of proteins</li>
<li>Usually stresses are too complex, too elusive or too quickly lethal to be handled by a well-regulated stress response.</li>
<li>selection will purge the elements in the population that are unable to find a qualified solution to the standing challenge.</li>
</ul></li>
<li>Mutators
<ul>
<li>Point mutations =&gt; slow generation of genetic novelty</li>
<li>Needs for increasing the mutation rates in times of stress (Bjedov et al., 2003)</li>
<li>substantial burden of deleterious mutations (Sniegowski et al., 2000)</li>
<li>when stresses are frequent and can be tackled by the generation of variability at a reduced number of loci: there is often selection for the maintenance of repeats engaging in recombination at those loci. In this case, repeats are under second-order selection because they generate at high rates the adaptive changes with which they hitchhike to fixation (Tenaillon et al., 2001). This does not suppose any sort of directionality; repeats will recombine randomly, resulting in sequence reassortment or gene conversion. When such events have a high probability of leading to adaptive changes, then the associated repeats will propagate in the population. On the other hand, repeats generating deleterious changes with a high frequency will in general be purged by the action of natural selection.</li>
</ul></li>
<li>Programmed hotspots of variation by recombination
<ul>
<li>associated with functions managing antagonistic social interactions (e.g., during arms races both partners are continuously changing to keep the other in check (Van Valen, 1973).)<br />
</li>
<li>Usually associated to negative frequency-dependent selection, i.e. a given variant is highly adaptive only when rare.</li>
</ul></li>
<li>Many forms of diversifying selection share the peculiarity that they hardly select for higher efficiency, but instead for sequences that are different while performing an equivalent function.</li>
</ul>
<h3 id="repeats-as-evolutionary-reservoirs">Repeats as evolutionary reservoirs</h3>
<ul>
<li>Homologous recombination between one (or a few) master gene and copies of it allows sequence variation without loss of function.</li>
<li>Repeats leading to recombination events with better results (i.e., diverse but functional variants), will be selected for.</li>
<li>This creates evolutionary reservoirs in the form of repeats scattered in the chromosome (Palmer &amp; Brayton, 2007).
<ul>
<li>e.g. Borrelia:
<ul>
<li>large number of plasmids containing the vast majority of repeated elements of the genome (Casjens, 1999).</li>
<li>Antigenic variation is under strong selection and proceeds by recombination processes within the repeats present in the plasmids (Norris, 2006).</li>
<li>antigenic variation results from replacement of internal segments of varied lengths of the master gene by gene conversion resulting from its recombination with the repeated elements (Zhang et al., 1997).</li>
<li>Replacements occur at high frequencies during infection, thereby allowing continuous escape from the immune system (McDowell et al., 2002).</li>
</ul></li>
</ul></li>
</ul>
<h3 id="transient-gene-inactivation">Transient gene inactivation</h3>
<p>Gene deletions may be highly adaptive in contexts of host–pathogen associations or for the adoption of new ecological niches (Sokurenko et al., 1999), even though they are expected to lower the cell fitness in the original habitat.</p>
<ul>
<li>Gene deletions may arise from recombination between repeats (Gaudriault et al., 2008)</li>
<li>Occurs at high rates among mismatch repair genes in enterobacteria (Bjedov et al., 2003) rich in close repeats (Rocha et al., 2002),</li>
<li>The loss of these genes increases mutations rates 100- to 1000-fold.</li>
<li>High load of deleterious changes, but in maladapted populations, the rate of adaptive mutations may be enough to allow mutators to increase in frequency (Taddei et al., 1997).</li>
<li>Adaptive for the new variant to revert to the nonmutator phenotype
<ul>
<li>This occurs at high frequencies because the loss of mismatch repair renders homologous recombination more permissive</li>
<li>Which facilitates the recovery of the intact genes by horizontal transfer.</li>
</ul></li>
<li>Losing a mismatch gene increases the likelihood of receiving one by horizontal transfer because recombination barriers are diminished. Most genes have no such feedback effect.</li>
<li>Strategy is limited to prokaryotes with high rates of horizontal gene transfer</li>
<li>Sexual isolation =&gt; gene deletion is irreversible. In this case, the larger the number of repeats in a gene, the more likely its irreversible deletion from the genome.</li>
</ul>
<h3 id="transient-gene-silencing">Transient gene silencing</h3>
<ul>
<li>Reversible way of function gain and loss: gene expression regulation (turned on or off under specific conditions).</li>
<li>Contingency loci are elements that are stochastically turned on and off resulting in random variation, phase variation, of the expression of the proteins under diversifying selection (Moxon et al., 1994) =&gt; Quasi species like: The population is now a distribution</li>
<li>Involve homologous recombination between one master gene and repeats in the chromosomes. In this case, recombination leads to gene inactivation or silencing. (Swanson et al., 1986).</li>
<li>Site-specific recombination systems are activated stochastically to produce chromosomal rearrangements leading to phase variation (Komano, 1999).</li>
<li>SSR (short repeated sequences in tandems) are less frequent than expected in prokaryotic genomes, especially inside genes (hence typically purged from genomes) (Field &amp; Wills, 1998; Ackermann &amp; Chao, 2006)</li>
<li>When present, they are often associated with genes under diversifying selection.</li>
<li>Important property of SSR is that changes in copy number are heritable and reversible. By subsequent contraction and expansions, it is possible for a gene to oscillate between expression and silencing. This is unattainable for deletions between spaced repeats, which require hazardous recovery of the missing sequence by horizontal transfer.</li>
<li>Gene deletion in mutS amplifies mutation rates in general, but more intensely so in SSR. This shows the tight associations that can be created within the different ways of generating variability in times of stress by means of recombination between repeats. While recombination between repeats may inactivate genes associated with mismatch repair, the loss of the latter leads to higher and more permissive recombina- tion processes that result in further generation of genetic diversity.</li>
</ul>
<h3 id="the-impact-of-selfish-dna">The impact of selfish-DNA</h3>
<p>Upon insertion, TEs may affect the expression of nearby genes, either by interrupting/silencing them or by enhancing their expression.</p>
<ul>
<li>TE elements also provide a structural basis to
<ul>
<li>duplications,</li>
<li>deletions,</li>
<li>rearrangements</li>
<li>incorporation of foreign DNA either by active transposition or by homologous recombination between the multiple copies present in a genome (Naas et al., 1995; Alokam et al., 2002; Kothapalli et al., 2005; Redder &amp; Garrett, 2006).</li>
<li>mediate the transfer of genetic information between genomes or between replicons of the same genome.</li>
<li>TE are believed to undergo frequent horizontal transfer and cycles of expansion and extinction within a given species, most likely as a consequence of transfer between genomes and plasmids (Wagner, 2006).</li>
<li>Important source of genomic diversity and plasticity (Schneider et al., 2002; Brugger et al., 2004; Nascimento et al., 2004; Yang et al., 2005).</li>
<li>genome size is to date the only statistically significant determinant of IS abundance (Touchon &amp; Rocha, 2007). Indeed, the numbers and density of IS increase rapidly with genome size, which is the exact inverse trend found for the density of genes under strong selection such as essential genes, suggesting that ISs are as strongly counter-selected as the average gene is under strong selection.</li>
</ul></li>
</ul>
<h3 id="transient-amplifications-and-amplicons">Transient amplifications and amplicons</h3>
<ul>
<li>two repeats separated by a spacer have the capacity to serve as amplicons, i.e., units that may amplify in tandem.</li>
<li>If the flanking repeats are long and distant, the amplification will engage homologous recombination.</li>
<li>If the repeats are close they will engage slipped-strand mispairing.</li>
<li>If they are both long and close they will engage both.</li>
<li>Doubts surrounding the relevance of such structures for the creation of stable paralogues</li>
<li>many results show that these structures accelerate adaptation by transient amplifications of genetic material (Anderson &amp; Roth, 1977).</li>
<li>In some rare cases, amplicons are extremely deleterious because they duplicate genetic elements that are needed in strictly single copies, but in the majority of cases they are neutral, i.e. without effect on fitness, beneficial or only mildly deleterious.</li>
<li>frequency of recombination depends on the repeat size and similarity, their relative distance, the natural selection of the ensuing amplification and the probability of recombination between such chromosomal locations.</li>
<li>Most amplicons are very stable and amplify at low rates. Yet, amplification produces large tandem repeats that recombine at high frequencies. The very large repeats resulting from amplifications are remarkably unstable, and homologous recombination quickly leads to either deletion or further amplification (Romero &amp; Palacios, 1997).</li>
<li>the rate-limiting step of the process corresponds to the initial amplification.</li>
<li>Tandem copied genes can occupy significant fractions of the chromosome (Jannie`re et al., 1985) (Smith et al., 1988).</li>
<li>Some amplicons may attain particularly high amplification numbers =&gt; rolling-circle replication (Petit et al., 1992).</li>
<li>It has even been proposed that clusters of functional neighboring genes, such as operons, are maintained to allow their coamplification (Reams &amp; Neidle, 2004).</li>
<li>plasmids may insert in the chromosome by duplication insertion, involving the production by homologous recombination of two copies of a repeat at the edges of the integrated element =&gt; structure of an amplicon. Elements integrated in this way are immediately ready to be further amplified (Haldenwang et al., 1980; Smith et al., 1988).</li>
<li>Laterally transferred genes are typically maladapted for expression on the host because their regulatory genes do not allow optimized expression. If they are amplified in the chromosome in high copy number, they may then adapt more rapidly by the joint action of three effects.
<ul>
<li>amplified regions offer a larger mutation target (more mutations accumulating in the amplified set of genes than in a single-gene locus).</li>
<li>recombination between repeats may engage the action of an error-prone polymerase and thus become locally mutagenic.</li>
<li>repeats will accumulate changes and allele reassortment may occur within chromosomes by recombination between the copies, accelerating the combination of favorable changes in one gene.</li>
</ul></li>
</ul>
<h2 id="the-fate-of-repeats">The fate of repeats</h2>
<ul>
<li>It is the fate of repeats to disappear.</li>
<li>This can take very diverse time frames depending on the repeats.</li>
<li>Genes under prolonged selection for gene dosage effects will last in genomes until such selection ceases.</li>
<li>Some repeats will fade away because they accumulate point mutations. Even if they may remain in genomes, they will end up by losing the ability to recombine and therefore cease to be considered as repeats.</li>
<li>Repeats carrying no significant lasting advantage will disappear quickly either by deletion or by counter-selection</li>
<li>Hence, while genomic rearrangement rates are close to genomic mutation rates, rearrangements are much more rarely fixed in populations than point substitutions because of their deleterious effects. Because the stability of genomes is largely a result of the rearrangements between repeats and ensuing natural selection, this means that:
<ul>
<li>selection for organization is extremely strong and the vast majority of rearrangements are purged (Rocha, 2006);</li>
<li>genomes with many repeats exhibit high rearrangement rates and are less fit regarding chromosome organization and gene repertoire stability (Rocha, 2003b)</li>
</ul></li>
<li>The size of genomes is largely the result of the balance of forces increasing genetic information, horizontal transfer and amplifications, and forces removing genetic information, selection and genetic deletions.</li>
<li>The genome will not increase too much since:
<ul>
<li>The limiting step in the formation of the amplicon is the creation of the flanking repeats. A deletion of these repeats will result in a sequence that will lead to a new amplicon at very low rates</li>
<li>large tandems of amplicons are expected to cause genetic instability and they create a severe imbalance in the gene dosage.</li>
<li>These will result in a deletion bias (Koch, 1979).</li>
</ul></li>
<li>Structure of repeats have different consequences (For the same number of repeats):
<ul>
<li>certain chromosomal distributions of the copies are less disruptive upon deletion:
<ul>
<li>close repeats will affect smaller fractions of the genome.</li>
<li>repeats tend to be distributed nonrandomly in genomes in such a way as to exert less negative effects on the chromosome upon recombination than expected by chance (Achaz et al., 2003).</li>
</ul></li>
</ul></li>
</ul>
<h1 id="turning-a-hobby-into-a-job"><a href="http://www.nature.com/nrg/journal/v9/n12/abs/nrg2482.html">turning a hobby into a job</a></h1>
<h2 id="which-genes-undergo-duplications">which genes undergo duplications:</h2>
<ul>
<li>Pattern that seems to hold only for small-scale duplications (SSDs), not for WGD16 (see below)
<ul>
<li>less densely clustered parts of the protein-interaction network =&gt; fewer pleiotropic constraints</li>
<li>Fewer pleiotropic constraints(14)</li>
<li>with smaller fitness defects when knocked out(15)</li>
</ul></li>
<li>indirect association between lower rates of evolution and essentiality (17)</li>
<li>Explantation:
<ul>
<li>Certain types of genes have biochemical features that allow them to be adapted easily to novel functions (substrate promiscuity)</li>
<li>Other types of genes might be particularly unlikely to undergo functional innovation via duplication, because the duplication has an immediate detrimental effect (so-called duplication-resistant genes(18)). =&gt; MOST SUPPORTED HYPOTHESIS
<ul>
<li>i.e., highly constrained cellular systems =&gt; selectively unfavourable is termed the dosage-balance hypothesis(1)</li>
</ul></li>
</ul></li>
</ul>
<h2 id="single-gene-duplications-versus-whole-genome-duplications">Single-gene duplications versus whole-genome duplications</h2>
<ul>
<li>Anticorrelation between genes fixed in duplicate after SGD and those surviving from WGD: Genes in a functional category are duplicated by SSD and not by WGD or vice versa. (25, 20)</li>
<li>WGD tend to diverge in expression more quickly (26)</li>
<li>Duplicates produced by WGD also seem to share more protein interactions after duplication than do genes duplicated by SSD (26, 27)</li>
<li>Enzymes retained in duplicate after WGD seem to have fitness defects at least as large as those for the genes that are not retained(16)</li>
<li>Dosage is not affected by WGD =&gt; WGD events might allow certain evolutionary novelties to appear and be selected for that would have been unlikely to arise otherwise.<br />
</li>
<li>WGD might also allow innovation through changes in relative gene dosage.
<ul>
<li>WGD may allow all the genes in a pathway to increase their relative dosages simultaneously, simply by remaining in duplicate as the rest of the genome shrinks</li>
</ul></li>
<li>WGD duplicate genes retain substantial degrees of functional overlap for long periods (31,32)</li>
<li>Many WGD loci that are fixed in duplicate in one yeast species are single-copy in others(33) =&gt; point to a high frequency of neofunctionalization.</li>
</ul>
<h3 id="the-nature-of-co-option">The nature of co-option</h3>
<ul>
<li>Co-option often occurs using some evolved feature of the gene in question, although perhaps in a surprising way (29, 34, 35)</li>
<li>Co-option can also result in proteins being used for functions that are seemingly unrelated to their evolved role (37, 38, 39)</li>
</ul>
<h3 id="natural-selection-and-co-option">Natural selection and co-option</h3>
<ul>
<li>In the absence of other factors, gene duplications are expected to be either selectively neutral(43) or detrimental(44, 46)</li>
<li>Three life stages of a duplicate gene pair:
<ul>
<li>creation (mutation in a single individual)</li>
<li>fixation–preservation (in the whole population by selection or drift)
<ul>
<li>Neofunctionalization:
<ul>
<li>New function on one member of a duplicate gene pair occurs after the duplication(51)</li>
<li>Gene appropriated for adaptation after duplication has a function that is closely related to the function required =&gt; novelty is built on a conserved enzymatic mechanism.</li>
</ul></li>
<li>Subfunctionalization:
<ul>
<li>Functions of an ancestrally multifunctional gene have become divided up neutrally among the daughter copies (degeneration, complementation model) (9, 37, 50)</li>
<li>escape from adaptive conflict (EAC) model (59–61): The two functions of the ancestral gene cannot simultaneously be optimized by natural selection. After gene duplication, the two daughter genes can escape from the conflict.</li>
<li>Several studies have suggested that the extant features of networks, such as those of protein–protein interactions and transcriptional regulation, are compatible with a node (gene) duplication model (64–66)</li>
</ul></li>
<li>Dosage
<ul>
<li>Duplicate gene pairs preserved by selection for increased gene dosage (70) =&gt; selectively advantageous at birth</li>
<li>Selection for higher gene dosage might increase the likelihood of a later neofunctionalization event by prolonging the half-life of a duplicate gene pair.</li>
</ul></li>
<li>Building new functions by dosage selection
<ul>
<li>A gene does not address only one function it also encode minor activities (78) (promiscuous protein activities)</li>
<li>These minor activities can also evolve into new functions: reservoir of functional novelty</li>
<li>Combine Selection for gene dosage and Neofunctionalization of minor activities : adaptive amplification (88,89) adaptive radiation(90) or IAD(45, 90) model
<ul>
<li>Duplications are selected in certain genes with weak but beneficial activities.</li>
<li>The tandem arrays of duplicates that this selection maintains have an increased rate of beneficial, activity-enhancing mutations owing to the large number of mutational targets.</li>
<li>Once such muta- tions have been fixed, selection to maintain the duplicated array is reduced and the extra copies are lost</li>
<li>leaving only the ancestral gene and the new duplicate with a novel function</li>
</ul></li>
</ul></li>
</ul></li>
<li>optimization</li>
<li>(loss of a duplication is by far the most common fate)</li>
</ul></li>
</ul>
<h1 id="ideas-for-program">Ideas for program:</h1>
<ul>
<li>Repeated elements mark the position for futur insertion of elements =&gt; higher probability to receive new TE or Horizontal Transfert sequences<br />
</li>
<li>Repeated elements are received through Horizontal gene transfert</li>
<li>It is possible to measure the essentiality of a gene (decrease of SAE it permits). So a possibility could be to have more chances to undergo duplications for non essential genes (at constant number of genes associated)</li>
<li>In order to respect the dosage effect hypothesis, the effect of duplication should be relative to the relative difference of number of genes in the same pathway (cluster). If only one gene of a cluster is duplicated =&gt; its position changes and not the other ones (e.g., twice further). I all genes are duplicated, the position of the core-points does not change.</li>
<li>Gene conversion =&gt; That is way we do not keep track of the sequence . . . usually it will be homogenized by gene conversion, until it escapes and tackles a new function</li>
<li>If the gene becomes too different (e.g., works for another centroid) we can consider that it becomes free to moove =&gt; less likelly to copy 2 genes of unrelated functions together.</li>
<li>Gene conversion from Horizontal gene transfert =&gt; We replace the location along a given dimension only considering points belonging to the core-point (and not very far away).</li>
<li>Amplification: various copies of same gene =&gt; weight matrix (less likelly to be at the origin of novelty)</li>
<li>Acquisition of new gene: Horizontal Gene Transfert =&gt; taking elements from the dataset (very likelly to be at the origin of novelty)</li>
<li>Another possibility to describe the mutation of existing genes (with various copies is the Homologous recombination between one (or a few) master gene and copies of it allows sequence variation without loss of function.)</li>
<li>To which function contribute Horizontal Gene transfert in probability?</li>
<li>Transient gene inactivation =&gt; It should be possible to give recently deleted genes/corepoints in Horizontal Gene Transfert from time to time. Simulaitng though that the gene is still in the population =&gt; Larger memory</li>
<li>Muators lead specially to more mutations on repeated parts</li>
<li>Genome size is positivelly correlated with the number of IS =&gt; More recombination =&gt; Bigger genomes will undergo more mutations. Small genomes rich on vital genes have few IS =&gt; less recombination</li>
<li>Amplicons are quite stables but the amplified sequences are very variable =&gt; have a weight of 1 &lt;=&gt; genome of amplicons and they are specially unstable when they have more copies. Rate limiting step in amplification are the initial steps. There are mechanisms that allow more rapid duplication (circular)</li>
<li>Laterally tranfered genes are usually maladapted and amplification can help them to be more adapted:</li>
</ul>
</body>
</html>
