<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
  <meta name="author" content="Sergio Peignier">
  <title>Sergio-Peignier-Home</title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<h1 id="genome-order">Genome order</h1>
<h2 id="selection-for-gene-clustering-by-tandem-duplication"><a href="http://www.annualreviews.org/doi/abs/10.1146/annurev.micro.58.030603.123806">SELECTION FOR GENE CLUSTERING BY TANDEM DUPLICATION</a></h2>
<p>Other reviews 61</p>
<h3 id="phenomenon-evidence">Phenomenon evidence</h3>
<ul>
<li>Bacterial genes with common function often reside within genetically linked groups:
<ul>
<li>Operons (87)</li>
<li>Clusters of functionally related genes often include more than a single operon</li>
<li>genes encoding physically interacting proteins do tend to be linked and their organization conserved in diverse organisms (12, 17, 21, 100)</li>
<li>Destruction of operons may be selectively neutral during long-term evolution (44)</li>
<li>However, layered over the observation that genomes are frequently shuffled and rearranged, highly conserved clusters have been identified (103)(58) […] gene order is not random (104)</li>
</ul></li>
</ul>
<h3 id="horizontal-transfert-model">Horizontal transfert model</h3>
<h4 id="phenomenon-evidence-1">Phenomenon evidence</h4>
<ul>
<li>Genomic island, describes regions of varied function […] Differences between islands and their neighboring regions and evidence of mobile genetic elements are indicative of horizontal gene transfer.</li>
<li>[In islands] Genetic signatures of mobility are evident, such as integrase genes and flanking direct repeats that are duplications of the insertion site.</li>
<li>Islands include large regions encoding variable physiological functions.</li>
<li>New nomenclature aims to emphasize that different chromosomal clusters are associated with alternative transfer mechanisms (e.g., conjugative transposons or integrative plasmids) (9, 107).</li>
<li>Gene clustering may originate as the result of horizontal gene transfer (61, 97).</li>
</ul>
<h4 id="model-59-61-97">Model (59, 61, 97)</h4>
<ul>
<li>According to this model, clustering can be thought of as a selfish property of the genes that allows the <strong>comobilization of a genetic region required for a common function (59).</strong></li>
<li><strong>Clustering facilitates the introduction of novel traits into different cells</strong>. selective benefit travels with the genes</li>
<li><strong>Clustered genes enable the step-wise construction of novel catabolic pathways by the reorganization of genetic segments (11, 86, 105, 111).</strong>
<ul>
<li><strong>Formation of the gene cluster:</strong>
<ul>
<li><strong>proximal genes more likely to be transferred</strong></li>
<li><strong>One function often involve severall genes</strong></li>
</ul></li>
<li><strong>Retention of transfered genes:</strong>
<ul>
<li><strong>when they provide benefit to the new recipient.</strong></li>
<li><strong>Genes repairment by homologus recombination of horizontal transfert: corepair (33)</strong></li>
</ul></li>
<li><strong>Retention of order:</strong>
<ul>
<li>Horizontal transfer: Shuffling of genes during evolution through rearrangements is not prevented by Horizontal transfert</li>
<li>Regulatory benefit of cotranscription is often cited as an obvious explanation for operons</li>
</ul></li>
</ul></li>
</ul>
<h3 id="retrograde-evolution-model-41">Retrograde Evolution model (41)</h3>
<h4 id="model">Model</h4>
<ul>
<li><strong>Enzymes in a metabolic pathway: recruited by duplication and divergence</strong></li>
<li>Creation of enzyme catalyzing new reaction and sharing the determinants used for substrate (product) binding. <strong>Adjacent genes in an operon would share a common ancestor and a related physiological purpose</strong></li>
<li>Proximal genes are rarely homologous (89): argument against retrograde evolution model.</li>
</ul>
<h3 id="coamplification-model">Coamplification model</h3>
<h4 id="model-1">Model</h4>
<ul>
<li>clustered genes to undergo <strong>tandem duplication</strong> and further <strong>amplification</strong> or <strong>deamplification</strong></li>
<li><strong>Rudimentary form of regulation:</strong>
<ul>
<li><strong>Reversible change of gene dosage</strong><br />
</li>
<li><strong>Single function may require the expression of multiple genes</strong></li>
<li><strong>Clustered genes: more likely to be coamplified and deamplified</strong></li>
<li><strong>Dynamic process can increase fitness by enabling adaptation to variable environments</strong></li>
<li><strong>Benefit should increase with proximity =&gt; clusters can form progressively</strong></li>
</ul></li>
<li>Increase <strong>adaptative mutation effect</strong>:
<ul>
<li><strong>High adaptive mutation rate reflects that there are actually many copies of the target gene (5, 36, 97)</strong></li>
<li><strong>Periods of relaxed selection (function not needed) =&gt; accumulation of detrimental mutations =&gt; mutations weaken the functionality of set of genes =&gt; coamplification increase gene dosage =&gt; help restore function.</strong></li>
<li><strong>Higher chance of reversion and/or the acquisition of compensatory mutations</strong></li>
<li>Reversible amplification allows clustered organization to be selected in preference to progeny where genetic rearrangement has dispersed the genes.</li>
</ul></li>
<li>evolution of <strong>novel functions and regulatory mechanisms:</strong>
<ul>
<li><strong>Gene duplications can place an operon in a new regulatory context. (e.g., operon fusion (4))</strong></li>
<li><strong>duplication = foundation for the acquisition and fixation of beneficial mutations (e.g., formation of paralogs with new or modified functions)</strong></li>
<li>Facilitates <strong>coupled duplication and divergence</strong>: Interacting proteins coevolve appears to result from the interrelated duplication of their genes (21, 54)**</li>
</ul></li>
</ul>
<h3 id="tandem-amplification-mechanism">Tandem Amplification mechanism</h3>
<ul>
<li>Multiple gene copies, often in a tandem head-to-tail array, allow cells to proliferate under growth-limiting conditions (4, 35, 95, 97)</li>
<li>Tandem duplication followed by further amplification (95, 97)</li>
<li>The initial duplication, which may involve unequal sister strand exchange, produces two adjacent identical copies of a region (the amplicon) that can undergo homologous (RecA-dependent) recombination to increase the number of copies.</li>
<li>Tandem arrays of identical sequences are unstable and can be deleted by homologous recombination.</li>
<li>Deamplification often occurs quickly after removal of the selective pressure or after the appearance of a mutation that alleviates the need for amplification (5, 95)</li>
<li>A high level of sequence identity in these regions increases the frequency of homologous recombination and can result in duplication hotspots (42, 95, 97)</li>
<li>All large direct repeated sequences have the potential to recombine and duplicate the intervening region (93)</li>
<li>Insertion sequences and mobile genetic elements clearly contribute to gene duplication by providing repeated sequences that can recombine (27, 64).</li>
</ul>
<h3 id="duplication-importance">Duplication importance</h3>
<ul>
<li>short-term reversible process of gene amplification should result in one of three long-term fates for the duplicated region:
<ul>
<li>selection, The selection of duplicate genes (paralogs) that have diverged is a fundamental method of obtaining new genes and novel functions (71). Thus, the evolutionary importance of gene duplication continues to be evaluated in light of new genome sequence data (40, 46, 51).</li>
<li>deletion,</li>
<li>silencing by mutational inactivation.</li>
</ul></li>
<li>According to the traditional view of paralog creation, one copy of a duplicated gene is redundant and free to be modified (71). This concept assumes that one copy is under selective pressure while the other can diverge freely (under neutral selection).</li>
<li>However, there appears to be no period of neutral evolution, and both paralogs evolve at similar rates (51). Kondrashov et al. (51) suggest that duplications preserved in an evolving lineage are immediately beneficial because of protein dosage effects in variable environments. This conclusion supports the basic tenet of the coamplification model that the tandem duplication of related genes can improve fitness (Figure 2).</li>
<li>genes acquired by horizontal transfer, which may also frequently undergo gene amplification, appear to be especially likely to result in paralog retention (39)</li>
</ul>
<h2 id="adaptative-amplification"><a href="http://www.tandfonline.com.sci-hub.cc/doi/full/10.1080/10409230701507757">Adaptative amplification</a></h2>
<ul>
<li>amplification occurs as part of a broad, general stress response =&gt; organisms have the capacity to induce structural changes in the genome:
<ul>
<li>heat stress tolerance, Resistance to insecticides, Resistance to antibiotics, Resistance to the herbicide, resistance to toxic metals</li>
<li>How can it activate this process?</li>
</ul></li>
<li>Amplification is seen as a device to extend the range of gene expression to handle the extreme conditions (Anderson and Roth, 1977; Roth et al., 1996; Romero and Palacios, 1997)</li>
<li>If the extreme conditions occur repeatedly: eventually the organism would acquire selected mutations that make the required amplification a more frequent event:
<ul>
<li>acquiring sufficient homology spanning the segment that is to be amplified for unequal homologous recombination to be able to form the initial duplication (Mekalanos, 1983)</li>
<li>evolving a site-specific recombination mechanism (Reams and Neidle, 2004b).</li>
<li>Amplification is reversible, readily breaking down by unequal homolo- gous recombination to a single copy when selection is relaxed.</li>
<li>The copies might diverge from each other or they might become separated by intervening sequences =&gt; cluster of like genes rather than a tandem array</li>
<li>Gene clustering: When related functions are clustered, they can be co-amplified so that all are upregulated and downregulated together.</li>
<li>Divergence:
<ul>
<li>reduce unequal homologous recombination (which changes the copy number)</li>
<li>reduce gene conversion, which tends to keep the repeated sequences alike.</li>
<li>Eventually, the clustered genes might become unlinked, which could reduce gene conversion further and allows evolution of new functions (Graham, 1995).</li>
<li>Ohno (1970): repetition of genes has a significant role in evolution.
<ul>
<li>One copy of a gene retains the original function,</li>
<li>The other is free to change without selection until it encodes a new or modified function.<br />
</li>
<li>while the repeated genes evolve, their domains can be reassorted by gene conversion,</li>
<li>eventually the most fit variant would replace the others by loss of copy number or by inactivation of the less fit copies, generating pseudogenes.</li>
<li>Processes might be facilitated by the <strong>increase in mutation rate per cell that is expected to accompany the multiple copies, on the assumption that each copy has the same mutation rate as a single copy</strong> (Roth et al., 1996).</li>
</ul></li>
</ul></li>
</ul></li>
</ul>
<h2 id="application-to-our-problem">Application to our problem</h2>
<ul>
<li>Idea: Instead of checking the distance in all dimensions, start comparing only one, then another . . . and so on excluding little by little options that do not work . . . Is it possible to build a simple regulation system (if a point in the sample with a value in this dimension exists then I turn on the candidate center)? . . . second element for ranking =&gt; time of computation</li>
<li>Proximal genes transfered together : new function acquired and/or correction through homologus recombination</li>
<li>Rudimentary form of regulation:
<ul>
<li>easy amplification/deamplification of set of genes =&gt; adaptation of phenotype to variable environment.</li>
<li>Acquiring or loosing genes =&gt; we should reevaluate the phenotype for these genes</li>
</ul></li>
<li>Adaptative mutations effect:
<ul>
<li>more genes =&gt; compensatory mutations =&gt; recovery of function (in place mutations at phenotype level)</li>
<li>mutated genes =&gt; not very effective =&gt; more of them =&gt; function achieved (same phenomenon than previous point)</li>
</ul></li>
<li>Source of novelty:
<ul>
<li>Coupled duplication and divergence =&gt; paralogs with new or modified functions.</li>
<li>Set of genes in new regulatory environment.</li>
</ul></li>
</ul>
<h3 id="possible-operations">Possible operations</h3>
<ul>
<li>Possible operations:
<ul>
<li>Tandem duplication =&gt; Increases weights matrix, leads to changes in the phenotype matrix to get closer to the optimum: we can only take pieces from points in this cluster.</li>
<li>Deamplification =&gt; Decreases the weights matrix, leads to changes in the phenotype matrix to get closer to the optimum: we can only take pieces from points in this cluster.</li>
<li>Horizontal transfert with homologus recombination =&gt; Replaces a set of elements in the phenotype matrix (the genes of donor should get to the corresponding cluster in receiver)</li>
<li>Horizontal transfert with new function acquisition =&gt; fills empty row in weights and phenotype matrices</li>
<li>Adaptative mutations =&gt; proportionnaly to weight we modify the phenotype matrix: we can only take pieces from points in this cluster.</li>
<li>New function acquisition =&gt; An existing row loses duplicates, mutates and a new row is filled with elements from the dataset. We can take directly a projection of a data point along the given dimensions.</li>
</ul></li>
<li>Idea for regulation:
<ul>
<li>Matrix of regulation. presence of point along this dim cose to this value =&gt; activates a set of candidate clusters.</li>
<li>We could also imagine to have the same but for inter-clusters. The expression of this cluster activates this other cluster.</li>
</ul></li>
</ul>
</body>
</html>
