# [SELECTION FOR GENE CLUSTERING BY TANDEM DUPLICATION](http://www.annualreviews.org/doi/abs/10.1146/annurev.micro.58.030603.123806)

+ clustered genes to undergo **tandem duplication** and further **amplification** or **deamplification**
+ **Rudimentary form of regulation:**
	+ **Reversible change of gene dosage**  
	+ **Single function may require the expression of multiple genes**
	+ **Clustered genes: more likely to be coamplified and deamplified**
	+ **Dynamic process can increase fitness by enabling adaptation to variable environments**
	+ **Benefit should increase with proximity => clusters can form progressively**
+ Increase **adaptative mutation effect**:
	+ **High adaptive mutation rate reflects that there are actually many copies of the target gene (5, 36, 97)**
	+ **Periods of relaxed selection (function not needed) => accumulation of detrimental mutations => mutations weaken the functionality of set of genes => coamplification increase gene dosage => help restore function.**
	+ **Higher chance of reversion and/or the acquisition of compensatory mutations**
	+ Reversible amplification allows clustered organization to be selected in preference to progeny where genetic rearrangement has dispersed the genes. 
+ evolution of **novel functions and regulatory mechanisms:**
	+ **Gene duplications can place an operon in a new regulatory context. (e.g., operon fusion (4))** 
	+ **duplication = foundation for the acquisition and fixation of beneficial mutations (e.g., formation of paralogs with new or modified functions)**
	+ Facilitates **coupled duplication and divergence**: Interacting proteins coevolve appears to result from the interrelated duplication of their genes (21, 54)**

+ Multiple gene copies, often in a tandem head-to-tail array, allow cells to proliferate under growth-limiting conditions (4, 35, 95, 97)
+ Tandem duplication followed by further amplification (95, 97)
+ The initial duplication, which may involve unequal sister strand exchange, produces two adjacent identical copies of a region (the amplicon) that can undergo homologous (RecA-dependent) recombination to increase the number of copies.
+ Tandem arrays of identical sequences are unstable and can be deleted by homologous recombination.
+ Deamplification often occurs quickly after removal of the selective pressure or after the appearance of a mutation that alleviates the need for amplification (5, 95)
+ A high level of sequence identity in these regions increases the frequency of homologous recombination and can result in duplication hotspots (42, 95, 97)
+ All large direct repeated sequences have the potential to recombine and duplicate the intervening region (93)
+ Insertion sequences and mobile genetic elements clearly contribute to gene duplication by providing repeated sequences that can recombine (27, 64).

+  short-term reversible process of gene amplification should result in one of three long-term fates for the duplicated region: 
	+ selection,  The selection of duplicate genes (paralogs) that have diverged is a fundamental method of obtaining new genes and novel functions (71). Thus, the evolutionary importance of gene duplication continues to be evaluated in light of new genome sequence data (40, 46, 51).
	+ deletion, 
	+ silencing by mutational inactivation.

+ According to the traditional view of paralog creation, one copy of a duplicated gene is redundant and free to be modified (71). This concept assumes that one copy is under selective pressure while the other can diverge freely (under neutral selection).
+ However, there appears to be no period of neutral evolution, and both paralogs evolve at similar rates (51). Kondrashov et al. (51) suggest that duplications preserved in an evolving lineage are immediately beneficial because of protein dosage effects in variable environments. This conclusion supports the basic tenet of the coamplification model that the tandem duplication of related genes can improve fitness (Figure 2).
+ genes acquired by horizontal transfer, which may also frequently undergo gene amplification, appear to be especially likely to result in paralog retention (39)


# [Genesis, effects and fates of repeats in prokaryotic genomes](http://femsre.oxfordjournals.org/content/33/3/539.abstract)

	+ homologous recombination between smaller or more degenerate repeats:
		+ Leads to gene conversion:
			+ increase the region of similarity between the elements
			+ Creates larger repeats (Santoyo & Romero, 2005)
			+ Maintains the sequence similarity between repetitive elements (Harvey & Hill, 1990) (Hughes, 2000).
		+ Some sequences escape from gene conversion:
			+ The sequence may diverges after a given threshold, it may become free from the homogenizing effect of gene conversion because recombination frequency drops exponentially 
	+ Illegitimate recombination between potentially very small and close repeats:
		+ Illegitimate recombination between small degenerate closely spaced 
			+ Cause dislocation mutagenesis => production of a strict repeat from a degenerate one (Schaaper, 1988).
			+ Given slipped-strand mispairing properties => repeats must be nearby in the genome but they may only be a few nucleotides long. Such repeats can be so small that they are easily found in genetic texts (Karlin & Cardon, 1994). 
		+ amplicon model of repeat generation (Romero & Palacios, 1997)
			+ First, illegitimate recombination leads to the creation of short close repeats (slippage events  (Edlund & Normark, 1981) ). 
			+ Second, these repeats are low-frequency amplicons that may amplify in tandem.
			+ Third, resulting amplifications are large and can expand and contract by homologous recombination (Levinson & Gutman, 1987)



+ most evolutionary novelty is thought to arise from the duplication and divergence of pre-existing elements (Ohno, 1970; Taylor & Raes, 2004).
+ Gene duplication creates redundancy, thus freeing the copies from some functional constraints. If the copies are effectively redundant, then one of them is likely to be lost by lack of selection against inactivating mutations
+ However, in rare cases, the accumulation of mutations may lead either to:
	+ neofunctionalization, one of the copies acquires a new function
	+ subfunctionalization, the two copies specialize in subfunctions of the primitive gene (Fig. 6) (Lynch & Katju, 2004)
	+ Both rely heavily on the accumulation of point mutations. 
+ If there is strong selection for both copies at the time of amplification, then the amplification will be stable and mutational processes could have much more time to diversify the function of the genes (Francino, 2005; Bergthorsson et al., 2007):
	+ e.g.,A gene confers a secondary function with weak efficiency 
	+ Need for that function increases,
	+ Amplification of the gene is selected
	+ amplified gene accumulates mutations in the different copies. 
	+ Mutations in one of the copies of the gene allowing an increase in efficiency of the secondary function become highly adaptive. 
	+ Other copies could be deleted or be used for the primary or other function 
	+ Leads to subfunctionalization (Francino, 2005). 
	+ Might be a frequent event (Patrick et al., 2007).
	+ Bias of deletion over amplification in prokaryotic genomes (Lawrence et al., 2001; Mira et al., 2001). Once repeats are created by recombination, typically in tandem, they are then deleted at high rates 
	+ gene conversion homogenize the two copies of the gene => reduce sequence divergence and slow subfunctionalization and neofunctionalization (Santoyo & Romero, 2005).
	+ large rearrangements tend to be highly deleterious (Rocha, 2008). Repeats can be stabilized if, after creation in tandem, they are physically separated by chromosome rearrangements (Achaz et al., 2000).
	+ CONCLUSION: ample evidence for gene amplifications in genomes, demonstrated by the abundance of repeated elements and some large families of paralogues (Snel et al., 2002; Gevers et al., 2004; Alm et al., 2006; Cho et al., 2007). It is unclear whether gene amplifications last long enough to be responsible for the creation of these large gene families (Lerat et al., 2005), but it is not clear...


+ Schematically, environmental challenges can be classified according to
	+ frequency with which they arise
	+ possibility of building appropriate physiological responses to tackle them.
	+ The evolutionary patterns associated with adaptation depends on both 
+ Stresses
	+ tackled by a physiological response based on the regulated expression of a set of proteins
	+ Usually stresses are too complex, too elusive or too quickly lethal to be handled by a well-regulated stress response.
	+ selection will purge the elements in the population that are unable to find a qualified solution to the standing challenge.
+ Mutators
	+ Point mutations => slow generation of genetic novelty
	+ Needs for increasing the mutation rates in times of stress (Bjedov et al., 2003) 
	+ substantial burden of deleterious mutations (Sniegowski et al., 2000)
	+ when stresses are frequent and can be tackled by the generation of variability at a reduced number of loci: there is often selection for the maintenance of repeats engaging in recombination at those loci. In this case, repeats are under second-order selection because they generate at high rates the adaptive changes with which they hitchhike to fixation (Tenaillon et al., 2001). This does not suppose any sort of directionality; repeats will recombine randomly, resulting in sequence reassortment or gene conversion. When such events have a high probability of leading to adaptive changes, then the associated repeats will propagate in the population. On the other hand, repeats generating deleterious changes with a high frequency will in general be purged by the action of natural selection.
+ Programmed hotspots of variation by recombination 
	+ associated with functions managing antagonistic social interactions (e.g., during arms races both partners are continuously changing to keep the other in check (Van Valen, 1973).)	
	+ Usually associated to negative frequency-dependent selection, i.e. a given variant is highly adaptive only when rare.
+ Many forms of diversifying selection share the peculiarity that they hardly select for higher efficiency, but instead for sequences that are different while performing an equivalent function.

+ Homologous recombination between one (or a few) master gene and copies of it allows sequence variation without loss of function.
+ Repeats leading to recombination events with better results (i.e., diverse but functional variants), will be selected for.
+ This creates evolutionary reservoirs in the form of repeats scattered in the chromosome (Palmer & Brayton, 2007). 
	+ e.g. Borrelia: 
		+ large number of plasmids containing the vast majority of repeated elements of the genome (Casjens, 1999).
		+ Antigenic variation is under strong selection and proceeds by recombination processes within the repeats present in the plasmids (Norris, 2006). 
		+ antigenic variation results from replacement of internal segments of varied lengths of the master gene by gene conversion resulting from its recombination with the repeated elements (Zhang et al., 1997). 
		+ Replacements occur at high frequencies during infection, thereby allowing continuous escape from the immune system (McDowell et al., 2002).

Gene deletions may be highly adaptive in contexts of hostâpathogen associations or for the adoption of new ecological niches (Sokurenko et al., 1999), even though they are expected to lower the cell fitness in the original habitat. 

+ Gene deletions may arise from recombination between repeats (Gaudriault et al., 2008) 
+ Occurs at high rates among mismatch repair genes in enterobacteria (Bjedov et al., 2003) rich in close repeats (Rocha et al., 2002),
+ The loss of these genes increases mutations rates 100- to 1000-fold. 
+ High load of deleterious changes, but in maladapted populations, the rate of adaptive mutations may be enough to allow mutators to increase in frequency (Taddei et al., 1997). 
+ Adaptive for the new variant to revert to the nonmutator phenotype
	+ This occurs at high frequencies because the loss of mismatch repair renders homologous recombination more permissive
	+ Which facilitates the recovery of the intact genes by horizontal transfer.
+ Losing a mismatch gene increases the likelihood of receiving one by horizontal transfer because recombination barriers are diminished. Most genes have no such feedback effect.
+ Strategy is limited to prokaryotes with high rates of horizontal gene transfer
+ Sexual isolation => gene deletion is irreversible. In this case, the larger the number of repeats in a gene, the more likely its irreversible deletion from the genome. 


+ two repeats separated by a spacer have the capacity to serve as amplicons, i.e., units that may amplify in tandem.
+ If the flanking repeats are long and distant, the amplification will engage homologous recombination.
+ If the repeats are close they will engage slipped-strand mispairing.
+ If they are both long and close they will engage both. 
+ Doubts surrounding the relevance of such structures for the creation of stable paralogues
+ many results show that these structures accelerate adaptation by transient amplifications of genetic material (Anderson & Roth, 1977). 
+ In some rare cases, amplicons are extremely deleterious because they duplicate genetic elements that are needed in strictly single copies, but in the majority of cases they are neutral, i.e. without effect on fitness, beneficial or only mildly deleterious.
+ frequency of recombination depends on the repeat size and similarity, their relative distance, the natural selection of the ensuing amplification and the probability of recombination between such chromosomal locations.
+ Most amplicons are very stable and amplify at low rates. Yet, amplification produces large tandem repeats that recombine at high frequencies. The very large repeats resulting from amplifications are remarkably unstable, and homologous recombination quickly leads to either deletion or further amplification (Romero & Palacios, 1997).
+ the rate-limiting step of the process corresponds to the initial amplification. 
+ Tandem copied genes can occupy significant fractions of the chromosome (Jannie`re et al., 1985) (Smith et al., 1988). 
+ Some amplicons may attain particularly high amplification numbers => rolling-circle replication (Petit et al., 1992). 
+ It has even been proposed that clusters of functional neighboring genes, such as operons, are maintained to allow their coamplification (Reams & Neidle, 2004).
+ plasmids may insert in the chromosome by duplication insertion, involving the production by homologous recombination of two copies of a repeat at the edges of the integrated element => structure of an amplicon. Elements integrated in this way are immediately ready to be further amplified (Haldenwang et al., 1980; Smith et al., 1988). 
+ Laterally transferred genes are typically maladapted for expression on the host because their regulatory genes do not allow optimized expression. If they are amplified in the chromosome in high copy number, they may then adapt more rapidly by the joint action of three effects.
	+ amplified regions offer a larger mutation target (more mutations accumulating in the amplified set of genes than in a single-gene locus).
	+ recombination between repeats may engage the action of an error-prone polymerase and thus become locally mutagenic.
	+ repeats will accumulate changes and allele reassortment may occur within chromosomes by recombination between the copies, accelerating the combination of favorable changes in one gene. 


+ It is the fate of repeats to disappear. 
+ This can take very diverse time frames depending on the repeats. 
+ Genes under prolonged selection for gene dosage effects will last in genomes until such selection ceases.
+ Some repeats will fade away because they accumulate point mutations. Even if they may remain in genomes, they will end up by losing the ability to recombine and therefore cease to be considered as repeats. 
+ Repeats carrying no significant lasting advantage will disappear quickly either by deletion or by counter-selection
+ Hence, while genomic rearrangement rates are close to genomic mutation rates, rearrangements are much more rarely fixed in populations than point substitutions because of their deleterious effects. Because the stability of genomes is largely a result of the rearrangements between repeats and ensuing natural selection, this means that:
	+ selection for organization is extremely strong and the vast majority of rearrangements are purged (Rocha, 2006); 
	+ genomes with many repeats exhibit high rearrangement rates and are less fit regarding chromosome organization and gene repertoire stability (Rocha, 2003b)
+ Structure of repeats have different consequences (For the same number of repeats):
	+ certain chromosomal distributions of the copies are less disruptive upon deletion: 
		+ close repeats will affect smaller fractions of the genome. 
		+ repeats tend to be distributed nonrandomly in genomes in such a way as to exert less negative effects on the chromosome upon recombination than expected by chance (Achaz et al., 2003).

