# Horizontal Gene Transfert 

## [SELECTION FOR GENE CLUSTERING BY TANDEM DUPLICATION](http://www.annualreviews.org/doi/abs/10.1146/annurev.micro.58.030603.123806)

#### Phenomenon evidence

+ Genomic island, describes regions of varied function [...] Differences between islands and their neighboring regions and evidence of mobile genetic elements are indicative of horizontal gene transfer.
+ [In islands] Genetic signatures of mobility are evident, such as integrase genes and flanking direct repeats that are duplications of the insertion site.
+ Islands include large regions encoding variable physiological functions. 
+ New nomenclature aims to emphasize that different chromosomal clusters are associated with alternative transfer mechanisms (e.g., conjugative transposons or integrative plasmids) (9, 107). 
+ Gene clustering may originate as the result of horizontal gene transfer (61, 97).

#### Model (59, 61, 97)
+ According to this model, clustering can be thought of as a selfish property of the genes that allows the **comobilization of a genetic region required for a common function (59).**
+ **Clustering facilitates the introduction of novel traits into different cells**. selective benefit travels with the genes
+ **Clustered genes enable the step-wise construction of novel catabolic pathways by the reorganization of genetic segments (11, 86, 105, 111).**
	+ **Formation of the gene cluster:**
		+ **proximal genes more likely to be transferred**
		+ **One function often involve severall genes**
	+ **Retention of transfered genes:**
		+ **when they provide benefit to the new recipient.**
		+ **Genes repairment by homologus recombination of horizontal transfert: corepair (33)**
	+ **Retention of order:**
		+ Horizontal transfer: Shuffling of genes during evolution through rearrangements is not prevented by Horizontal transfert
		+ Regulatory benefit of cotranscription is often cited as an obvious explanation for operons



## [Genesis, effects and fates of repeats in prokaryotic genomes](http://femsre.oxfordjournals.org/content/33/3/539.abstract)

+ horizontal gene transfer:
	+ creation of repeats whenever the incoming DNA contains elements similar to information already present in the genome.
	+ integration into genome by:
		+ site-specific recombination (e.g., phage integrase)
		+ Homologus recombination:
			+ if  incoming DNA contains homology with the chromosome at both extremities => double cross-over leads to genetic replacement (Niaudet et al., 1985).
			+ if incoming DNA is circular => then one single region of similarity with the chromosome allows integration of the genetic element by homologous recombination in a Campbell-like manner (Duncan et al., 1978). =>  repeats are created close together as they are spaced by the sequence of the incoming element.


+ Horizontal Transfert:
	+ quick avenue for acquiring functions that have already endured selection in other contexts
	+ ample evidence that it brings new genes into most genomes at very high rates (Hao & Golding, 2006)

+ Adaptive for the new variant to revert to the nonmutator phenotype
	+ This occurs at high frequencies because the loss of mismatch repair renders homologous recombination more permissive
	+ Which facilitates the recovery of the intact genes by horizontal transfer.

Upon insertion, TEs may affect the expression of nearby genes, either by interrupting/silencing them or by enhancing their expression.

+ TE elements also provide a structural basis to
	+ duplications, 
	+ deletions, 
	+ rearrangements
	+ incorporation of foreign DNA either by active transposition or by homologous recombination between the multiple copies present in a genome (Naas et al., 1995; Alokam et al., 2002; Kothapalli et al., 2005; Redder & Garrett, 2006).
	+ mediate the transfer of genetic information between genomes or between replicons of the same genome. 
	+ TE are believed to undergo frequent horizontal transfer and cycles of expansion and extinction within a given species, most likely as a consequence of transfer between genomes and plasmids (Wagner, 2006). 
	+ Important source of genomic diversity and plasticity (Schneider et al., 2002; Brugger et al., 2004; Nascimento et al., 2004; Yang et al., 2005). 
	+ genome size is to date the only statistically significant determinant of IS abundance (Touchon & Rocha, 2007). Indeed, the numbers and density of IS increase rapidly with genome size, which is the exact inverse trend found for the density of genes under strong selection such as essential genes, suggesting that ISs are as strongly counter-selected as the average gene is under strong selection.

+ Laterally transferred genes are typically maladapted for expression on the host because their regulatory genes do not allow optimized expression. If they are amplified in the chromosome in high copy number, they may then adapt more rapidly by the joint action of three effects.
	+ amplified regions offer a larger mutation target (more mutations accumulating in the amplified set of genes than in a single-gene locus).
	+ recombination between repeats may engage the action of an error-prone polymerase and thus become locally mutagenic.
	+ repeats will accumulate changes and allele reassortment may occur within chromosomes by recombination between the copies, accelerating the combination of favorable changes in one gene. 

+ plasmids may insert in the chromosome by duplication insertion, involving the production by homologous recombination of two copies of a repeat at the edges of the integrated element => structure of an amplicon. Elements integrated in this way are immediately ready to be further amplified (Haldenwang et al., 1980; Smith et al., 1988). 
